pom.xml

```diff
@@ -18,6 +18,10 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-actuator</artifactId>
```

Comment:
👍 for this
is it a nice to have or is it required for K8s (for K8s probes, I'd assume?)

Reply:
need it for the probes